CFN/EC-lz-Config-Guardduty-all-regions.yml

```diff
@@ -136,58 +128,9 @@ Resources:
       State: ENABLED
       Targets:
       - Id: "CrossAccountTargetId"
-        Arn: !Sub "arn:aws:events:${AWS::Region}:${SecLogMasterAccountId}:event-bus/default"
+        Arn: !Sub "arn:aws:events:eu-west-1:${SecLogMasterAccountId}:event-bus/default"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

ARN in Resource GuardDutyRuleComplianceChangeEvent contains hardcoded Partition in ARN or incorrectly placed Pseudo Parameters